test/export_import_test.py

```diff
@@ -0,0 +1,40 @@
+import shutil, unittest, logging, os
+import config.root # Set source directory
+
+from unittest.mock import patch
+from subprocess import call
+
+log = logging.getLogger("Test Import Export")
+logging.basicConfig(format='%(asctime)s %(name)s:%(message)s', level=logging.DEBUG)
+
+class Test(unittest.TestCase):
+    @patch('logging.Logger.error')
+    def test_export_import_profile_data(self, mock_error):
+        # Get current working directory
+        current_dir = os.getcwd()
+        # Change working directory to extras to run script and export data
+        try:
+            os.chdir("../simulators/extras") # IDE
+        except:
+            os.chdir("simulators/extras") # Command line
+
+        # Run the ExportProfileData.py script
+        call(["python", "ExportProfileData.py"])
+
+        # Check if the export_data folder is created
+        self.assertTrue(os.path.exists("export_data"), msg="export_data folder not found")
+
+        # Run the ImportData.py script and get the exit code
+        exit_code = call(["python", "ImportData.py"])
```

Comment:
- before importing, we should remove all MEAs from at least one profile (with externalID sim_001_profile) so that we can check after the import step if any new MEA exists for this profile